app/controllers/api/v1/mc_questions_controller.rb

```diff
@@ -17,15 +17,28 @@ def create
           return
         end
 
-        options[:options].each do |option|
-          @mc_question.mc_options << McOption.new(option: option)
+        begin
+          find_survey(mc_question_params[:survey_id])
```

Comment:
I guess the survey will be made live before we attempt to add questions? We will have to keep track of the order of API calls.

Reply:
@AndrewHall797 not sure what you mean

Reply:
oh sorry ignore this